CMakeLists.txt

```diff
@@ -220,6 +220,12 @@ if (UNBUNDLED OR NOT (CMAKE_SYSTEM MATCHES "Linux" OR APPLE) OR ARCH_32)
     option (NO_WERROR "Disable -Werror compiler option" ON)
 endif ()
 
+if (CMAKE_SYSTEM MATCHES "FreeBSD")
+    set (NOT_FREEBSD 0)
```

Comment:
This option should be named `USE_TCMALLOC_BY_DEFAULT`.
Current name has no sense.